src/io.rs

```diff
@@ -508,7 +508,7 @@ mod test {
     fn read_write_read_matrix_market_via_csc() {
         let path = "data/matrix_market/simple.mm";
         let mat = read_matrix_market::<f64, usize, _>(path).unwrap();
-        let csc = mat.to_csc();
+        let csc = mat.to_csc::<usize>();
```

Comment:
An alternative to the turbofish syntax would be to constrain the type of `csc`, something like:

```rust
let csc: crate::CsMat = mat.to_csc();
```

For some reason I tend to prefer this syntax, but it's only a matter of taste.

Reply:
If you would like me to switch to that syntax just let me know.

Reply:
Never mind, I'm switching